tests/e2e/note-images.test.ts

```diff
@@ -0,0 +1,154 @@
+import fs from 'fs'
+import { faker } from '@faker-js/faker'
+import { expect, test } from '@playwright/test'
+import { type NoteImage, type Note } from '@prisma/client'
+import { prisma } from '#app/utils/db.server.ts'
+import { insertedUsers, loginPage } from '#tests/playwright-utils.ts'
+
+test('Users can create note with an image', async ({ page }) => {
+	const user = await loginPage({ page })
+	await page.goto(`/users/${user.username}/notes`)
+
+	const newNote = createNote()
+	const altText = 'cute koala'
+	await page.getByRole('link', { name: 'new note' }).click()
+
+	// fill in form and submit
+	await page.getByRole('textbox', { name: 'title' }).fill(newNote.title)
+	await page.getByRole('textbox', { name: 'content' }).fill(newNote.content)
+	await page.setInputFiles(
+		"input[name='images[0].file']",
```

Comment:
Is there another way to locate this input field? This feels like an implementation detail.

Reply:
One solution is to use `getByLabel` but for multiple inputs, all of them have the same label `Image` which isn't ideal, so if the implementation is changed to have different labels for each image selection input then they can be selected using these unique labels.
Also, I think these multiple inputs should have different labels to make it easy for the screen readers.

Another solution is to not touch labels but have unique but predictable testids for each file input which can be selected using `getByTestId`.

Let me know how you want to approach this.

<img width="1512" alt="Screenshot 2023-09-01 at 10 31 36 AM" src="https://github.com/epicweb-dev/epic-stack/assets/61059807/994e493e-7272-4b3b-9e25-383628f29473">

Reply:
We can use the `nth` method on`getByLabel` just like I mentioned for textarea

Reply:
Yeah, I'd like to completely avoid complicated locator strings. Could you do that here too?

Reply:
fixed here https://github.com/epicweb-dev/epic-stack/pull/420/commits/edae2f0f6efaa432b1d63963ebf3ddf502e8706c